registry/storage/driver/testsuites/testsuites.go

```diff
@@ -143,7 +158,10 @@ func (suite *DriverSuite) TestInvalidPaths(c *check.C) {
 
 	for _, filename := range invalidFiles {
 		err := suite.StorageDriver.PutContent(suite.ctx, filename, contents)
-		defer suite.StorageDriver.Delete(suite.ctx, firstPart(filename))
+		// only delete if file was succesfully written
```

Comment:
I think we need this, otherwise if `err == nil` we'll leave the file around and abort the entire test suite on test teardown.